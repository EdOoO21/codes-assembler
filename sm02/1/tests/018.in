-749 354 -81 -315 -510 -595 949 2 286 11 -626 -18 -395 -62 816 -459 -599 -481 841 416 -759 -336 69
