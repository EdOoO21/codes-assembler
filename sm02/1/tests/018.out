69
-336
-759
416
841
-481
-599
-459
816
-62
-395
-18
-626
11
286
2
949
-595
-510
-315
-81
354
-749
